content/ansible-tower-advanced/8-isolated-nodes/_index.md

```diff
@@ -164,12 +164,14 @@ Go to **Templates** in the **RESOURCES** section of the menu, click the ![plus](
 
 - **CREDENTIAL:** Example Credentials
 
-- **INSTANCE GROUPS:** `dmz`
```

Comment:
It's unnecessary because indeed the inventory is already using the DMZ instance group. For me, it's more confusing why you should set the instance group on both inventory and job template, and we've explained before that you can set the instance group at different levels, but we can discuss this through on Monday.

Reply:
thanks for fixing it @ericzolf